module/Finna/src/Finna/View/Helper/Root/Config.php

```diff
@@ -82,4 +82,15 @@ public function isGeographicMapVisible(): bool
     {
         return !empty($this->get('facets')->Geographical->map_selection);
     }
+
+    /**
+     * Display similar records at the bottom of record view
+     * as a carousel
+     *
+     * @return bool
+     */
+    public function similarRecordsCarouselEnabled(): bool
+    {
+        return !empty($this->get('config')->Record->similar_carousel);
```

Comment:
Käytä null coalescing -operaattoria (ja oikeaa asetuksen nimeä), niin on selkeämpää, mikä on oletus:
`return $this->get('config')->Record->similar_carousel_bottom ?? false;`
tai asetuksen toiminnan muutoksen jälkeen:
`return $this->get('config')->Record->similar_carousel_display ?? '';`